### PR TITLE
Fix issue #39 - Added code to the js sendRequest with that will allow…

### PR DESCRIPTION
**Pull request by alfrye**
… request to xtext services to be CORS enabled

Signed-off-by: alfrye <alfrye@gmail.com>

### CODE REVIEW

**Comment by cdietrich** [comment]
@spoenemann @holgerschill can you please have a look

File: org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js [hunk 1]
```diff
@@ -164,7 +164,11 @@ define(['jquery'], function(jQuery) {
 	XtextService.prototype.sendRequest = function(editorContext, settings, needsSession) {
 		var self = this;
 		self.setState('started');
-		
+		var corsEnabled = editorContext._editor.options['enableCors'];
```
Comment (spoenemann) [inline org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js]: Use `editorContext.xtextServices.options` to retrieve the options.

File: org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js [hunk 2]
```diff
@@ -164,7 +164,11 @@ define(['jquery'], function(jQuery) {
...
+		if(corsEnabled !== undefined && corsEnabled) {
```
Comment (spoenemann) [inline org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js]: Redundant condition: just write `if (corsEnabled)` (`undefined` is evaluated to `false`)

Comment (alfrye) [reply org.eclipse.xtext.web/src/main/js/xtext/services/XtextService.js]: Thanks! I thought that was the case. I will make that change.

**Pull request merged**